launch/cli/endpoints.py

```diff
@@ -251,3 +253,44 @@ def edit_endpoint(ctx: click.Context, endpoint_name: str):
         # TODO: Print out a nice error message if the user passes in arguments
         # that fail server-side validation.
         client.edit_model_endpoint(model_endpoint=model_endpoint, **kwargs)
+
+
+@endpoints.command("send")
+@click.argument("endpoint_name")
+@click.option("-r", "--request", help="input request as a json string")
+@click.option("--json-file", help="json file containing request")
+@click.pass_context
+def send(
+    ctx: click.Context,
+    endpoint_name: str,
+    request: Optional[str],
+    json_file: Optional[str],
+):
+    """Sends request to launch endpoint"""
+
+    # Only allowed one kind of input
+    assert (request is not None) ^ (
+        json_file is not None
+    ), "Please supply EITHER --request OR --json-file"
+
+    if request is not None:
+        json_input = json.loads(request)
+    elif json_file is not None:
+        with open(json_file, "rb") as f:
+            json_input = json.load(f)
+
+    client = init_client(ctx)
+
+    model_endpoint = client.get_model_endpoint(endpoint_name)
+    print(f"Sending request to {endpoint_name=} at {ctx.obj.gateway_endpoint}")
+    if model_endpoint.status() is None:
+        raise ValueError(f"Unable to find endpoint {endpoint_name}")
+
+    if model_endpoint.status() != "READY":
+        print(f"Warning: endpoint is not ready get: {model_endpoint.status()}")
+    else:
+        future = model_endpoint.predict(
+            request=EndpointRequest(args=json_input, return_pickled=False)
+        )
+        response = future.get()  # blocks until completion
+        print(response)
```

Comment:
This assumes the endpoint is asynchronous - it would be awesome if you could handle the case for sync / async endpoints

Reply:
what's the syntax like for a sync endpoint?